spk/sabnzbd/Makefile

```diff
@@ -14,7 +14,7 @@ DESCRIPTION_SPN = SABnzbd hace que Usenet sea lo más simple posible, automatiza
 ADMIN_PORT = 8080
 RELOAD_UI = yes
 DISPLAY_NAME = SABnzbd
-CHANGELOG = "Enable sc-download group"
+CHANGELOG = "Update to 1.0.0. Note - When you upgrade an existing 0.7.x installation, release 1.0.0 will NOT be able to finish the queue. It's best to empty your queue first using 0.7.x. Alternatively, you can install 1.0.0 and then use queue repair on the Status page. Some information (like the order of the queue) will be lost."
```

Comment:
Use this format: `"1. Update to 1.0.0<br>2. xyz"`
The `<br>` adds a line break when shown in Package Center and shows that there are two distinct entries in the log.